_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -439,77 +465,69 @@ that are not covered by the automated scripts. If you wish to update further, fi
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/ecs-service-with-alb.adoc[ecs-service-with-alb Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/eks.adoc[EKS Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/iam-cross-account.adoc[iam-cross-account Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/iam-groups.adoc[iam-groups Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/iam-user-password-policy.adoc[iam-user-password-policy Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/jenkins.adoc[Jenkins Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/kms-master-key.adoc[kms-master-key Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/memcached.adoc[Memcached Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/openvpn-server.adoc[OpenVPN Server Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/rds.adoc[RDS Service migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/redis.adoc[Redis Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/route53-private.adoc[Route 53 (private) Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/route53-public.adoc[Route 53 (public) Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/sns-topics.adoc[sns-topics Service Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/vpc-app.adoc[VPC (app) Migration Guide]
+* link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/blob/master/_docs/_ref_arch_v1_to_v2_migration_guides/vpc-mgmt.adoc[VPC (mgmt) Migration Guide]
```

Comment:
Reminder to update this to release tag instead of `master` when we cut release.

Reply:
We can do this now if I can get feedback on https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/427